### PR TITLE
Update keystore-service.md

### PR DESCRIPTION
**Pull request by chadandycha**

### CODE REVIEW

File: murano/tutorials/keystore-service.md [hunk 1]
```diff
@@ -9,17 +9,17 @@ A common need when managing a fleet of devices is to track metrics and error log
...
+Murano solutions provide a [datapoint event](../../services/device/#datapoint) that gives you an opportunity to respond to incoming device data. When handling datapoints you may perform data conversion, store data, or send alerts. You may modify the way your solution responds to data by editing the handler for the datapoint event in Murano's web-based code editor or by uploading code using the command line tool. For the purpose of this tutorial, let's use the web-based editor. To modify the datapoint event handler, click on the SERVICES tab of your solution, select Products, and select the code tab. The code found here is executed when any devices with product types associated with the solution send data. You can now add code to store metrics to the Keystore service.
 
 ### Increasing a daily metric counter
```
Comment (chadandycha) [inline murano/tutorials/keystore-service.md]: Just want to be sure: was the word "atomically" intended here?

"It does so atomically, so you won't miss any incoming data points."

File: murano/tutorials/keystore-service.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ Keystore.command({ command = "lpush", key = "logs", args = {message}})
 ```
 
 We then keep only the last logs items with the trimming command by specifying a range of data to keep. Note that the list index starts at 0.
```
Comment (chadandycha) [inline murano/tutorials/keystore-service.md]: Is "last logs items" correct, or should it be "last log items"?

Comment (dweaver) [inline murano/tutorials/keystore-service.md]: Yes, "last log items"

File: murano/tutorials/keystore-service.md (hunk 1 above)
Comment (dweaver) [inline murano/tutorials/keystore-service.md]: Yes, use of "atomically" is intended here.

**Pull request merged**